_author_instructions/05_tutorials.md

```diff
@@ -7,8 +7,12 @@ excerpt: What is a publishable tutorial?
 permalink: /authors/tutorials/
 ---
 
-## What is a publishable tutorial?
-Tutorials are articles that take users through specific tasks or sequences of tasks to help the users learn how to do these tasks on their own for other applications or purposes. 
+## What is a publishable tutorial and/or training article?
+As detailed below, a tutorial is largely a procedural guide to a particular piece of software, whereas a training article provides an introduction to a procedure or theory and thus provides general background preparation for performing molecular simulation calculations with diverse software packages.
+The guidelines below will discuss tutorials and training articles separately, but the editors recognize that authors may wish to submit hybrid training/tutorial articles.  Authors pursuing a hybrid approach should carefully lay out their proposal in a pre-submission letter.  The editors are not setting pre-conditions on hybrid articles, except that they should combine elements from both types of papers in a logical way suitable to the goals of the article.
+
+## Tutorials
```

Comment:
Maybe "Tutorials: Definition and scope" to read parallel to the training section?

Reply:
sure, makes sense